scripts/container-startup.sh

```
@@ -16,4 +16,4 @@ python manage.py clearsessions
 # Add custom commands here
 
 echo 'Starting Django Server...'
-python manage.py runserver 0.0.0.0:80 
\ No newline at end of file
+gunicorn tcf_core.wsgi:application --bind 0.0.0.0:80"
```

Shellcheck (0.11.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)